yt_dlp/extractor/mediasite.py

```diff
@@ -26,7 +26,7 @@
 
 
 class MediasiteIE(InfoExtractor):
-    _VALID_URL = r'(?xi)https?://[^/]+/Mediasite/(?:Play|Showcase/(?:default|livebroadcast)/Presentation)/(?P<id>%s)(?P<query>\?[^#]+|)' % _ID_RE
+    _VALID_URL = r'(?xi)https?://[^/]+/Mediasite/(?:Play|Showcase/[^/\#\?]+/Presentation)/(?P<id>%s)(?P<query>\?[^#]+|)' % _ID_RE
```

Comment:
1. I am not familiar with mediasite, so sorry if this is a dumb question...
Isn't this regex too general? Especially since there is no domain defined. Is there no list of known types that we can use instead of `default|livebreoadcast`?

2. Shouldn't the regex be `[^/#?]`? The escaping is unnecessary afaik

Reply:
The linked issue has an example where this fragment is `vala2020`. To me, this clearly indicates this is not a ‘type’ at all, but an open-ended field.

The escaping may indeed be unnecessary. I was worried the `#` may be misinterpreted as a comment character (given the `(?x)`).